### PR TITLE
Swope known issue

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

**Review by dislbenn — APPROVED** [review]
LGTM

File: release_notes/known_issues_network.adoc [hunk 1]
```diff
@@ -30,6 +30,25 @@ For more about deprecations and removals, see xref:../release_notes/deprecate_re
...
 
+See the following known issues and limitations that might occur while using networking features.
+
+[#cluster-management-addon-needed]
+=== Without _ClusterManagementAddon_ submariner add-on fails
+//2.8:24159
+
+For versions 2.8 and earlier, when you install {product-title-short}, you also deploy the `submariner-addon` component with the Operator Lifecycle Manager. If you did not create a `MultiClusterHub` custom resource, the `submariner-addon` pod sends an error and prevents the operator from installing. 
+
+The following occurs because the `ClusterManagementAddon` custom resource definition is missing:
```
Comment (cadawson) [inline release_notes/known_issues_network.adoc]: The following notification occurs...?

File: release_notes/known_issues_network.adoc [hunk 2]
```diff
@@ -30,6 +30,25 @@ For more about deprecations and removals, see xref:../release_notes/deprecate_re
...
+
+----
+graceful termination failed, controllers failed with error: the server could not find the requested resource (post clustermanagementaddons.addon.open-cluster-management.io)
+----
+
+The `ClusterManagementAddon` resource is created by the `cluster-manager` deployment, however, this deployment becomes available when the `MultiClusterEngine` components are installed on the cluster. 
+
+When the `MultiClusterHub` custom resource is created, if there is not a `MultiClusterEngine` resource that is already available on the cluster, the `MultiClusterHub` operator deploys the `MultiClusterEngine` instance and the operator that is required, resolving the previous error.
```
Comment (cadawson) [inline release_notes/known_issues_network.adoc]:
```suggestion
If there is not a `MultiClusterEngine` resource that is already available on the cluster when the `MultiClusterHub` custom resource is created,  the `MultiClusterHub` operator deploys the `MultiClusterEngine` instance and the operator that is required, which resolves the previous error.
```

**Review by cadawson — APPROVED** [review]
Looks good! A couple of suggestions for consideration. Thanks!

**Pull request merged**